DiscordCardLinker/CardBot.cs

```diff
@@ -130,19 +160,19 @@ public class CardBot {
 			 * Only add the card key once.
 			 */
 			if(!collection.ContainsKey(key)) {
-				Console.WriteLine("Adding Key: ["+key+"]");
+				//Console.WriteLine("Adding Key: ["+key+"]");
 				collection.Add(key, new List<CardDefinition>());
 			}
 
-			Console.WriteLine("  .. ID.........: "+card.ID);
-			Console.WriteLine("  .. ImageURL...: "+card.ImageURL);
-			Console.WriteLine("  .. WikiURL....: "+card.WikiURL);
-			Console.WriteLine("  .. CollInfo...: "+card.CollInfo);
-			Console.WriteLine("  .. DisplayName: "+card.DisplayName);
-			Console.WriteLine("  .. Title......: "+card.Title);
-			Console.WriteLine("  .. Subtitle...: "+card.Subtitle);
-			Console.WriteLine("  .. TitleSuffix: "+card.TitleSuffix);
-			Console.WriteLine("  .. Nicknames..: "+card.Nicknames);
+			//Console.WriteLine("  .. ID.........: "+card.ID);
```

Comment:
Please do not comment out debugging information. This information is critical for debugging the container service if it is in a frozen state.